Bugzilla/CGI.pm

```diff
@@ -594,11 +594,11 @@ sub send_cookie {
     }
 
     # Add the default path and the domain in.
-    $paramhash{'-path'} = Bugzilla->params->{'cookiepath'};
-    $paramhash{'-domain'} = Bugzilla->params->{'cookiedomain'}
-        if Bugzilla->params->{'cookiedomain'};
+    my $uri = URI->new(Bugzilla->localconfig->{urlbase});
+    $paramhash{'-path'} = $uri->path;
+    $paramhash{'-domain'} = $uri->host;
```

Comment:
since urlbase won't change, you could do the URI parsing once and cache instead of with every request